src/content/docs/webdev-hosting/Netlify/netlify-case-study.md

```diff
@@ -0,0 +1,107 @@
+
+- Git interegation: As mentioned earlier another core feature for Netlify is the Git interegation which lets developers deploy sites directly from platforms like GitHub, GitLab, and Bitbucket.
+- Collaboration and Team Workflow: Netlify also has a few features that are also really great for team collaborations and workflow such as deployment previews and role-based access control.
+
+- Continuous Deployment: Netlify has a feature that supports continuous deployment, by
+  connecting your repository, Netlify will automatically build, deploy and update the site whenever changes are made.
+
+- Analytics and Monitoring: This feature is really awesome it lets you track your monitor traffic, site performance and gives you good insight into user behavior. Its essetiantly a tool that´ll help developers get better performance, optimazation and user experience.
+
+## Strengths
+
+- Ease of Use: As said ealier, Netlify is already known for its user-friendly interface and workflows. With just a few clicks developers can easily deploy their projects with Git intergration.
+
+- Serverless Functions: With Serverless Functions it gives developers the ability to execute server-side logic without managing infrastructure.
+
+- Continuous Deployment: Netlify supports continuous deployment, by
+  connecting your repository, Netlify will automatically build, deploy and update the site whenever changes are made.
+
+- Developer-Friendly: Netlify is very user-friendly and really easy to dive in too with all the documentation, tutorials and resources that are available for developers. And by doing this it lets developers get started quickly and makes troubleshooting a lot easier aswell.
+
+## Weaknesses
+
+- Cost: Even though Netlify is very known for being really good and generous with its free basic features, with that being said the costs can get really high for organzations that may have a lot of traffic and so because of this additional costs for premium features may occur.
+
+- Complexity for Large-Scale Applications: Since Netlify is more suited for small applications, you may find it lack more advanced features and options for large scale
+  applications. Beacuse of this organzations may choose other options for hosting instead.
+
+- Limited Backend: As stated earlier Netlify relies on serverless functions, because of this it may limit amount of backend customization and control available to developers.
+
+- Third-Party Services: Third-Party Services is a big part of netlify, it relies on services like AWS and and CDN services. The big risk with this is that this may introduce dependencies which then makes uptime and reliability affected.
+
+## Market Comparison
+
+If compared to a more flexible and advanced hosting website like Heroku, I would say that Netlify excels and shines the most when it comes to hosting static websites and fast deployment while offering great features such as Git Intergrations, Forms Handling and serverless functions.
```

Comment:
Consider rephrasing this using a third-person form. As it is, it is not objective and universal enough.
```suggestion
When compared to a more flexible and advanced hosting website like Heroku, Netlify excels and shines the most in hosting static websites and fast deployment while offering great features such as Git integrations, forms handling, and serverless functions.
```

Reply:
> Consider rephrasing this using a third-person form. As it is, it is not objective and universal enough.

This is the fix that i pushed, is this fine? https://github.com/Sugal99/development-platforms/commit/39db174a6ce78ad04a0af12f31ee85644e50819c